test/vanilla/AcceptanceTests/number_tests.py

```diff
@@ -59,24 +59,13 @@ def test_numbers(self):
         client.number.put_small_double(2.5976931e-101)
         client.number.put_big_double_negative_decimal(-99999999.99)
         client.number.put_big_double_positive_decimal(99999999.99)
-        client.number.put_big_decimal(Decimal(2.5976931e+101))
```

Comment:
Those tests made 0 sense.
- you were the **only** one with those tests (not even C# with native `decimal` had them)
- the testserver (since it is written in Node) can't handle numbers from the `decimal` range, which is (I guess), why those numbers here where taken from the double test - but then, WTF is the point...